default/scripting/ship_parts/ShortRange/SR_WEAPON_1_1.focs.txt

```diff
@@ -4,6 +4,7 @@ Part
     class = ShortRange
     damage = 3
     NoDefaultCapacityEffect
+    totalFighterDamageEstimation = 0
```

Comment:
why is this 0?

Reply:
A mass driver destroys exactly zero fighters per battle.

Reply:
maybe this needs a clearer name then... like `maxFightersDestroyedPerBattleEstimate` (though that's a bit long...)

Reply:
made it destroyFightersPerBattleMax and damageStructurePerBattleMax